testing/openshift-common.sh

```diff
@@ -64,19 +64,33 @@ yell() { echo "$0: $*" >&2; }
 
 die() {
   yell "$*"
-  destroy_crc && exit 111
+  exit 111
 }
 
 try() { "$@" || die "cannot $*"; }
 
 function setup_crc() {
+  # Set bundle
```

Comment:
Looks like these scripts are not shfmt'ed. Not a hard requirement but better to do.

Reply:
shfmt applied!